containers/podman-desktop
# feat: Initial Welcome page

### What does this PR do?

Creates a new Welcome page that will be shown once for every user. As per the feature this is just a simple page for now, but could be expanded in future sprints to help the user install podman or do other first-time setup.

Implementation notes:
- A welcome.version configuration property (not shown in settings) is used to store the fact that the welcome page has been shown. The value is set to 'initial' for now. My intention was that this could be set to the version of podman desktop or welcome page shown so in a future sprint we can decide when people who have seen the welcome should now see a newer/better version.
- The new page is an overlay that appears only once, based on the configuration.
- I've added the podman desktop logo as images/DesktopIcon because I don't believe we've used it elsewhere, but will likely need it on other pages in the future.
- The welcome page has a blurred background image and a static list of supported container engines. I've added a link to Settings and made a few other minor changes based on feedback from previous draft PR #1566.

### Screenshot/screencast of this PR

<img width="990" alt="Screenshot 2023-03-16 at 12 10 23 PM" src="https://user-images.githubusercontent.com/19958075/225682776-e016ba31-5bb9-41d5-87ed-f93caeb3fb84.png">

### What issues does this PR fix or reference?

Issue #1334.

### How to test this PR?

The welcome should appear the first time you start and then never again. To see the welcome a second time, remove the welcome.version property from the configuration (e.g. .local/share/containers/podman-desktop/configuration/settings.json on Mac) or delete the config entirely.
